src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/Methodologies/MethodologyController.cs

```diff
@@ -21,53 +20,53 @@ public MethodologyController(IMethodologyService methodologyService)
             _methodologyService = methodologyService;
         }
 
-        [Produces("application/json")]
-        [HttpGet("api/topic/{topicId}/methodologies")]
-        public Task<ActionResult<List<MethodologyViewModel>>> GetTopicMethodologiesAsync([Required]Guid topicId)
-        {
-            return _methodologyService
-                .GetTopicMethodologiesAsync(topicId)
-                .HandleFailuresOrOk();
-        }
-        
         [Produces("application/json")]
         [HttpGet("api/methodologies")]
-        public Task<ActionResult<List<MethodologyViewModel>>> GetMethodologiesAsync()
+        public Task<ActionResult<List<MethodologyStatusViewModel>>> GetMethodologiesAsync()
         {
             return _methodologyService
                 .ListAsync()
                 .HandleFailuresOrOk();
         }
-        
+
         [Produces("application/json")]
-        [ProducesResponseType(typeof(MethodologyViewModel), 200)]
+        [ProducesResponseType(typeof(MethodologyTitleViewModel), 200)]
         [ProducesResponseType(typeof(ValidationProblemDetails), 400)]
         [HttpPost("api/methodologies")]
-        public Task<ActionResult<MethodologyViewModel>> CreateMethodologyAsync(
-            CreateMethodologyViewModel methodology)
+        public Task<ActionResult<MethodologyTitleViewModel>> CreateMethodologyAsync(
+            CreateMethodologyRequest methodology)
         {
             return _methodologyService
                 .CreateMethodologyAsync(methodology)
                 .HandleFailuresOrOk();
         }
+
+        [Produces("application/json")]
+        [HttpGet("api/methodology/{methodologyId}/status")]
+        public async Task<ActionResult<MethodologyStatusViewModel>> GetMethodologyStatusAsync(Guid methodologyId)
+        {
+            return await _methodologyService
+                .GetStatusAsync(methodologyId)
+                .HandleFailuresOrOk();
+        }
         
         [Produces("application/json")]
         [HttpGet("api/methodology/{methodologyId}/summary")]
-        public async Task<ActionResult<MethodologyViewModel>> GetMethodologySummaryAsync(Guid methodologyId)
+        public async Task<ActionResult<MethodologySummaryViewModel>> GetMethodologySummaryAsync(Guid methodologyId)
         {
             return await _methodologyService
-                .GetAsync(methodologyId)
-                .HandleFailuresOr(Ok);
+                .GetSummaryAsync(methodologyId)
+                .HandleFailuresOrOk();
         }
-        
+
         [Produces("application/json")]
         [HttpPut("api/methodology/{methodologyId}/status")]
-        public async Task<ActionResult<MethodologyViewModel>> UpdateMethodologyStatusAsync(Guid methodologyId, 
+        public async Task<ActionResult<MethodologyStatusViewModel>> UpdateMethodologyStatusAsync(Guid methodologyId,
```

Comment:
As we'll need an endpoint to update the methodology, I would suggest removing this endpoint as it kind of becomes redundant.

Reply:
Ok, if you are sure @ntsim because I thought this is currently used on the tab titled "Release status" when approving for publication with a Release note.

Reply:
Yeh I would say get rid of it. I would also suggest we should get rid of the methodology `/status` endpoint too! Funnily enough, the frontend doesn't even get the methodology status using the `/status` endpoint.

I think this is a great example of where we haven't really been following RESTful API design. A methodology status isn't really something that's actually decoupled from a methodology, and having it like this only really serves to add extra endpoints for no reason.

Reply:
Cool, let's get rid of it. What about this though?

`  getMethodologyStatus: (methodologyId: string): Promise<MethodologyStatus> =>
    client
      .get<BasicMethodology>(`/methodology/${methodologyId}/status`)
      .then(methodology => methodology.status),
`

Reply:
I had renamed `/summary` on the front end to `/status` and introduced a new back end endpoint to get just the status as that was all it was using. It's been removed now in favour of the new `/summary`.